src/network/MessagePublisher.js

```diff
@@ -0,0 +1,44 @@
+const exchangeName = 'fog';
+const exchangeControl = 'control';
+
+export default class MessagePublisher {
```

Comment:
I'd prefer to move the `export` expression to the end of the file in order to follow the pattern of the other classes.